publish/working-with-pipelines/working-with-pipelines.md

```diff
@@ -309,6 +309,76 @@ You can also see pods for the pipeline runs, for which you can specify `oc descr
 
 If the pipeline run was successful, you can see a Docker image in our Docker registry and a pod that’s running your application.
 
+## Transport layer security (TLS) verification for image registry access in pipelines
```

Comment:
Can we move this whole section up in the document below the `Using stacks that are published to internal and private registries in pipelines` section?

Reply:
Updated as requested